File: back/DevC/src/main/java/com/back/devc/domain/interaction/postLike/dto/LikedPostResponse.java
package com.back.devc.domain.interaction.postLike.dto;

import java.time.LocalDateTime;

public record LikedPostResponse(
        Long postId,
        String title,
        String authorNickname,
        long likeCount,
        long commentCount,
        long viewCount,
        LocalDateTime createdAt,
        boolean liked,
        boolean bookmarked
) {
}

File: back/DevC/src/main/kotlin/com/back/devc/domain/interaction/postlike/dto/LikedPostResponse.kt
package com.back.devc.domain.interaction.postLike.dto

import java.time.LocalDateTime

data class LikedPostResponse(
    val postId: Long,
    val title: String,
    val authorNickname: String,
    val likeCount: Long,
    val commentCount: Long,
    val viewCount: Long,
    val createdAt: LocalDateTime,
    val liked: Boolean,
    val bookmarked: Boolean,
)